sdk/data/azcosmos/example_test.go

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azcosmos_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos"
+)
+
+// This example shows you how to get started using the Azure Cosmos DB SDK for Go.
+// NewCosmosClient creates a new instance of CosmosClient with the specified values. It uses the default pipeline configuration.
+func Example() {
+
+	endpoint, _ := os.LookupEnv("SOME_ENDPOINT")
+	key, _ := os.LookupEnv("SOME_KEY")
+
+	// Create new Cosmos DB client.
+	cred, _ := azcosmos.NewSharedKeyCredential(key)
+	client, err := azcosmos.NewCosmosClient(endpoint, cred, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// All operations for Go operate on a context.Context, allowing you to control cancellation/timeout.
+	ctx := context.Background()
+
+	// ===== 1. Creating a database =====
+
+	databaseName := azcosmos.CosmosDatabaseProperties{Id: "databaseName"}
+	database, err := client.CreateDatabase(ctx, databaseName, nil, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// This example showcases several common operations to help you get started, such as:
+	// ===== 2. Creating a container =====
+
+	properties := azcosmos.CosmosContainerProperties{
+		Id: "aContainer",
+		PartitionKeyDefinition: azcosmos.PartitionKeyDefinition{
+			Paths: []string{"/myPartitionKey"},
+		},
+	}
+
+	throughput := azcosmos.NewManualThroughputProperties(400)
+
+	resp, err := database.DatabaseProperties.Database.CreateContainer(ctx, properties, throughput, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	container := resp.ContainerProperties.Container
+
+	// ===== 3. Update container properties =====
+
+	resp, err = container.Read(ctx, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	updatedProperties := azcosmos.CosmosContainerProperties{
+		Id: "aContainer",
+		PartitionKeyDefinition: azcosmos.PartitionKeyDefinition{
+			Paths: []string{"/myPartitionKey"},
+		},
+		ETag: "someEtag",
+		IndexingPolicy: &azcosmos.IndexingPolicy{
+			IncludedPaths: []azcosmos.IncludedPath{},
+			ExcludedPaths: []azcosmos.ExcludedPath{},
+			Automatic:     false,
+			IndexingMode:  azcosmos.IndexingModeNone,
+		},
+	}
+
+	// Replace container properties
+	resp, err = container.Replace(ctx, updatedProperties, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Read the manual throughput property
+	throughputResponse, _ := container.ReadThroughput(ctx, nil)
+	_, err = throughputResponse.ThroughputProperties.ManualThroughput()
```

Comment:
Can we actually show the printing of the value for both this and the read of the AutoscaleMaxThroughput? Otherwise it's not really showing any usage of the value.

```
manualThroughput, err := throughputResponse.ThroughputProperties.ManualThroughput()
if err != nil {
    log.Fatal(err)
}
fmt.Printf("Container is provisioned with %v RU/s", manualThroughput)
```